src/thunderbots/software/ai/passing/evaluation.cpp

```diff
@@ -75,7 +75,7 @@ double AI::Passing::ratePassShootScore(const Field& field, const Team& enemy_tea
     Angle open_angle_to_goal =
         angleSweepCircles(pass.receiverPoint(), field.enemyGoalpostNeg(),
                           field.enemyGoalpostPos(), obstacles, ROBOT_MAX_RADIUS_METERS)
-            .second;
+            ->second;
```

Comment:
We aren't checking the optional is valid before accessing the value

Reply:
changed to:
```
    auto shot_opt = angleSweepCircles(pass.receiverPoint(), field.enemyGoalpostNeg(),
                                   field.enemyGoalpostPos(), obstacles, ROBOT_MAX_RADIUS_METERS);
    Angle open_angle_to_goal = Angle::zero();
    if (shot_opt){
        open_angle_to_goal = shot_opt->second;
    }

```